Calibration/EcalAlCaRecoProducers/plugins/BuildFile.xml

```diff
@@ -8,4 +8,15 @@
 <use   name="DataFormats/TrackReco"/>
 <use   name="CommonTools/UtilAlgos"/>
 <use   name="DataFormats/RecoCandidate"/>
+# for FED list selector
+<use   name="DataFormats/EgammaCandidates"/>
+<use   name="CalibFormats/SiStripObjects"/>
+<use   name="CalibCalorimetry/HcalAlgos"/>
+<use   name="CondFormats/EcalObjects"/>
+<use   name="CondFormats/DataRecord"/>
+<use   name="CondFormats/HcalObjects"/>
+<use   name="CondFormats/SiPixelObjects"/>
+<use   name="Geometry/CaloGeometry"/>
+<use   name="RecoEcal/EgammaCoreTools"/>
+<use   name="TrackingTools/TrackAssociator"/>
 <flags   EDM_PLUGIN="1"/>
```

Comment:
was trying to figure out why all of these dependencies came in. Could you clean up the header file for the moved plugin to use only the incldues that it needs (which is almost none of them- use "class blah;" instead) and add the minimal set of headers to the cc file.